### ISSUE TITLE
Constistent shortname for autotroph Si Uptake diagnostic

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#22]
Most autotroph diagnostics have a shortname of

```
sname = trim(autotrophs(n)%sname) // '_ABCDEF'
```

for some value of `ABCDEF`, but Si Uptake omits the `_` and sets

```
sname = trim(autotrophs(n)%sname) // 'bSi_form'
```

**Comment by mnlevy1981** [comment]
While this is a very simple fix, we need to inform the user community that the variable will have a different name in POP history files before implementing

**Comment by mnlevy1981** [comment]
closed via #186
